scripts/001-simple-web-server/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM node:18-alpine3.19
-COPY . /app
+RUN adduser -D app
+USER app
 WORKDIR /app
+COPY 001-simple-web-server.js constants.js package.json app/
```

Comment:
I moved `COPY` below `WORKDIR` because changes in the files would otherwise invalidate the cache in subsequent layers.